game/webcam.hh

```diff
@@ -12,16 +13,15 @@ namespace cv {
 }
 
 struct CamFrame {
-	int width;
-	int height;
+	int width = 0;
+	int height = 0;
 	std::vector<boost::uint8_t> data;
```

Comment:
instead of `std::vector<boost::uint8_t>` use `std::vector<std::uint8_t>` Don't forget the include `#include <cstdint>`